### PR TITLE
Fixed pip install - III

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description
* Unpinned dependencies
* Added `-U` and removed `-q` flags from pip install

__Related to:__ `fix test errors`

__Any expected test failures?__
* Tests on python3.11 might fail due to version incompatibilities

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [ ] adds a new feature
- [X] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [X] Refactored code ([sourcery](https://sourcery.ai/))
- [ ] Tested code locally
- [X] Precommit installed and run before pushing changes
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

File: integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb [hunk 1]
```diff
@@ -1027,7 +1031,9 @@
    },
    "outputs": [],
    "source": [
-    "run[\"confusion-matrix\"] = npt_utils.create_confusion_matrix_chart(rfc, X_train, X_test, y_train, y_test)"
+    "run[\"confusion-matrix\"] = npt_utils.create_confusion_matrix_chart(\n",
+    "    rfc, X_train, X_test, y_train, y_test\n",
```
Comment (Blaizzy) [inline integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb:1035]: Can you strip this extra space?

File: integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb [hunk 2]
```diff
@@ -944,7 +944,11 @@
    "source": [
     "import neptune.new.integrations.sklearn as npt_utils\n",
     "\n",
-    "run = neptune.init_run(project=\"common/sklearn-integration\", api_token=neptune.ANONYMOUS_API_TOKEN, name=\"other-options\")"
+    "run = neptune.init_run(\n",
+    "    project=\"common/sklearn-integration\",\n",
```
Comment (Blaizzy) [inline integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb:948]: Can you strip this extra space?

**Review by Blaizzy — APPROVED** [review]
LGTM! 

Just need to strip spaces from 2 lines in the sklearn notebook

File: integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb (hunk 1 above)
Comment (SiddhantSadangi) [reply integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb:1035]: this is indentation
image

File: integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb (hunk 2 above)
Comment (SiddhantSadangi) [reply integrations-and-supported-tools/sklearn/notebooks/Neptune_Scikit_learn.ipynb:948]: this is indentation

**Pull request merged**